solutions/common-shared-flows/correlation-v1/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "correlation-v1",
+  "version": "1.0.0",
+  "scripts": {
+    "deploy": "npm i && npx apigeetool deploySharedflow -u $APIGEE_USER -p $APIGEE_PASS -o $APIGEE_ORG -e $APIGEE_ENV -n $npm_package_name -V",
```

Comment:
Have we landed on the decision of this being the recommended deployment method throughout?

Reply:
you mean apigeetool vs maven?

Reply:
yes - applied-apigee uses apigeetool